File: src/main/java/br/edu/ufsj/tecweb/pigman/dtos/PigDTO.java
package br.edu.ufsj.tecweb.pigman.dtos;

import java.util.Date;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PigDTO {

    private Date lastVerfimuge;
    private String sex;
    private Float weight;
    private Date entryDate;
    private Date birthDate;
    private Long stallId;

}

File: frontend/src/app/dtos/pig.dto.ts
export interface outputPigDto{
  id: number;
}

export interface inputPigDto {
  id: number;
  sex: string;
  lastVermifuge: Date;
  weight: number;
  entryDate: Date;
  birthDate: Date;
  stallName: string;
  stallId: number;
}
